test/integration/attachment_access_limited_integration_test.rb

```diff
@@ -0,0 +1,71 @@
+require 'test_helper'
+require 'capybara/rails'
+
+class AttachmentAccessLimitedIntegrationTest < ActionDispatch::IntegrationTest
+  extend Minitest::Spec::DSL
+  include Capybara::DSL
+  include Rails.application.routes.url_helpers
+
+  let(:organisation) { create(:organisation) }
+  let(:managing_editor) { create(:managing_editor, organisation: organisation, uid: 'user-uid') }
+
+  before do
+    login_as managing_editor
+  end
+
+  context 'given a draft document with file attachment' do
+    let(:edition) { create(:news_article, organisations: [organisation]) }
+
+    before do
+      setup_publishing_api_for(edition)
+      publishing_api_has_linkables([], document_type: "topic")
```

Comment:
Minor: I know that strictly speaking this isn't specific to an edition, but I might be tempted to move this into the `setup_publishing_api_for` method.

Reply:
I'll try to remember to take a look at this when I revisit the code.